mm2src/coins/z_coin/z_rpc.rs

```diff
@@ -257,8 +243,12 @@ impl ZRpcOps for NativeClient {
                 header: Vec::new(),
                 vtx: compact_txs,
             };
-            on_block(compact_block)?;
+            db.insert_block(compact_block.height as u32, compact_block.encode_to_vec())
```

Comment:
Instead of doing direct cast from u64 to u32, can you handle the overflow?

Reply:
This is still not fixed.